addons/website_blog/views/website_blog_templates.xml

```diff
@@ -487,5 +495,55 @@
          <li><a href="#" data-action="new_blog_post">New Blog Post</a></li>
     </xpath>
 </template>
+
+<!-- RSS Feed -->
+<template id="website_blog_rss" name="Blog RSS">&lt;?xml version="1.0" encoding="utf-8"?&gt;
```

Comment:
As this template is totally independant from the website builder, can you put it in another file?